### ISSUE TITLE
[Bug] Kissmanga not working "HTTP Error 403"

### ISSUE THREAD

**Issue by sk-dark** [inorichi/tachiyomi#2756]
### Device information
* Tachiyomi version: 0.8.5
* Android version: 9

## Steps to reproduce
1. Update or download or open any manga from kissmanga. 

Or
Enter kissmanga from catalogues.

### Expected behavior
Manga opens

### Actual behavior
Shows "HTTP Error 403"

![Screenshot_2020-03-24-20-05-46-235_eu kanade tachiyomi](https://user-images.githubusercontent.com/59594284/77434657-65cf4700-6e0b-11ea-9a85-d3deea4a248f.jpg)
![Screenshot_2020-03-24-20-05-55-977_eu kanade tachiyomi](https://user-images.githubusercontent.com/59594284/77434738-81d2e880-6e0b-11ea-9f63-267f33d13900.jpg)

**Comment by CarlosEsco** [comment]
https://github.com/inorichi/tachiyomi-extensions/issues/2496

**Comment by Kuroiju** [comment]
I have been experiencing the same problems too. I need help on how to fix this

**Comment by HaraDesu** [comment]
you all, please read FAQ.

TL;DR: try open it in webview, how? click catalogue -> click browse on source that you desire  -> click that three dot -> click open in webview.


also, please do read FAQ, because this is a super duper common unavoidable problem. although, the dev team sure can make it less appear.

**Comment by Kuroiju** [comment]
> you all, please read FAQ.
> 
> TL;DR: try open it in webview, how? click catalogue -> click browse on source that you desire -> click that three dot -> click open in webview.
> 
> also, please do read FAQ, because this is a super duper common unavoidable problem. although, the dev team sure can make it less appear.

I did the instructions u gave but when I submit the code it resets again back to captcha verifying if Im a bot or not

**Comment by mrcn0** [comment]
> > you all, please read FAQ.
> > TL;DR: try open it in webview, how? click catalogue -> click browse on source that you desire -> click that three dot -> click open in webview.
> > also, please do read FAQ, because this is a super duper common unavoidable problem. although, the dev team sure can make it less appear.
> 
> I did the instructions u gave but when I submit the code it resets again back to captcha verifying if Im a bot or not

Same here

**Comment by sk-dark** [comment]
> > > you all, please read FAQ.
> > > TL;DR: try open it in webview, how? click catalogue -> click browse on source that you desire -> click that three dot -> click open in webview.
> > > also, please do read FAQ, because this is a super duper common unavoidable problem. although, the dev team sure can make it less appear.
> > 
> > 
> > I did the instructions u gave but when I submit the code it resets again back to captcha verifying if Im a bot or not
> 
> Same here

Me too

**Comment by HaraDesu** [comment]
> > > > you all, please read FAQ.
> > > > TL;DR: try open it in webview, how? click catalogue -> click browse on source that you desire -> click that three dot -> click open in webview.
> > > > also, please do read FAQ, because this is a super duper common unavoidable problem. although, the dev team sure can make it less appear.
> > > 
> > > 
> > > I did the instructions u gave but when I submit the code it resets again back to captcha verifying if Im a bot or not
> > 
> > 
> > Same here
> 
> Me too

i honestly don't care. i am just a person who willing to help. not a god that can wand my stick and make your problem disappear.

**Comment by sk-dark** [comment]
> > > > > you all, please read FAQ.
> > > > > TL;DR: try open it in webview, how? click catalogue -> click browse on source that you desire -> click that three dot -> click open in webview.
> > > > > also, please do read FAQ, because this is a super duper common unavoidable problem. although, the dev team sure can make it less appear.
> > > > 
> > > > 
> > > > I did the instructions u gave but when I submit the code it resets again back to captcha verifying if Im a bot or not
> > > 
> > > 
> > > Same here
> > 
> > 
> > Me too
> 
> i honestly don't care. i am just a person who willing to help. not a god that can wand my stick and make your problem disappear.

Dude chill...
No ones asking you to be god just don't give advice without testing it out first or is it working for you??

**Comment by HaraDesu** [comment]
> > > > > > you all, please read FAQ.
> > > > > > TL;DR: try open it in webview, how? click catalogue -> click browse on source that you desire -> click that three dot -> click open in webview.
> > > > > > also, please do read FAQ, because this is a super duper common unavoidable problem. although, the dev team sure can make it less appear.
> > > > > 
> > > > > 
> > > > > I did the instructions u gave but when I submit the code it resets again back to captcha verifying if Im a bot or not
> > > > 
> > > > 
> > > > Same here
> > > 
> > > 
> > > Me too
> > 
> > 
> > i honestly don't care. i am just a person who willing to help. not a god that can wand my stick and make your problem disappear.
> 
> Dude chill...
> No ones asking you to be god just don't give advice without testing it out first or is it working for you??

![image](https://user-images.githubusercontent.com/32463794/77453434-d8e7b600-6e29-11ea-91b2-7690a3e56f42.png)

**Comment by joseph619** [comment]
Don't quote each other, it's making the thread to hard to read.

Also, Kissmanga website is down for now. You have to be patient until it's up again.

**Comment by lshaf** [comment]
About kissmanga,
What's different between .com and .org?
